
ocp-point:
  horizon_length: 100
  wStateReg: 0.02
  wControlReg: 0.001
  wLimit: 1000
  wPCoM: 500
  wVCoM: 0
  wWrenchCone: 0
  wGripperPos: 5
  wGripperRot: 30
  wGripperVel: 2
  stateWeights: [
    500, 500, 500, 1000, 1000, 1000, 500, 500, 500, 500, 1000,
    1000, 500, 500, 500, 500, 1000, 1000, 100, 200, 100, 100, 100, 100, 1, 1,
    1, 500, 500, 500, 500, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
    1, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10
  ]
  controlWeights: [
    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
  ]
  

mpc-point:
  T_initialization: 100
  T_stabilization: 200
  T_drilling: 100
  targetPos: [0.6,0.4,1.1]
  holes_offsets: [
    [0,0,0]]
  #   [-0.078,0,0],
  #   [-0.078,-0.045,0],
  #   [0,-0.045,0]
  # ]
  backwardOffset: 0.05
  tolerance: 0.005
  maxGoalWeight: 100
  toolFramePos: [0, -0.02, -0.0825]
  use_mocap: 0
  use_gainScheduling: 1
  gainSchedulig_slope: 0.5


